web/index.php

```diff
@@ -30,9 +30,14 @@
 // Instantiate a Slim app
 $app = new Slim(array(
     'log.enabled' => true,
-    'templates.path' => '../templates'
+    'templates.path' => __DIR__.'/../templates'
```

Comment:
Pushed all of your changes except the "__DIR__." line above to my heroku instance, I receive a 200 and the expected video. Can you double check that this is in fact required?